src/components/AgendaItems/Create/AgendaItemsCreateModal.tsx

```
@@ -152,185 +154,188 @@ const AgendaItemsCreateModal: React.FC<
-          <Form.Group className="mb-3">
-            <Form.Label>{t('attachments')}</Form.Label>
-            <Form.Control
-              accept="image/*, video/*"
-              data-testid="attachment"
-              name="attachment"
-              type="file"
-              id="attachment"
-              multiple={true}
-              onChange={handleFileChange}
+        <div className="mb-3">
+          <label className="form-label">{t('url')}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)